packages/zql-integration-tests/src/chinook/planner.pg.test.ts

```diff
@@ -0,0 +1,100 @@
+import {beforeAll, describe, expect, test} from 'vitest';
```

Comment:
I'll add more cases here after we add a debug delegate and tracing of planner execution.

See `planner-controlled.test.ts` for more exhaustive tests.